src/services/users/users.ts

```diff
@@ -157,17 +157,25 @@ export default (
       );
     },
 
-    async requestPasswordReset(email, options) {
-      return (
-        (
-          await userClient.get(http, '/forgot_password', {
-            ...options,
-            params: {
-              email,
-            },
-          })
-        ).status === Results.Success
-      );
+    async requestPasswordReset(data: string | PasswordResetRequestData, options) {
+      const clientOptions = {
+        ...options,
+        params: {},
+      };
+
+      if (typeof data === 'string') {
+        clientOptions.params.email = data;
+      } else {
+        clientOptions.params.email = data.email;
+
+        if (data.mode) {
+          clientOptions.params.mode = data.mode;
+        }
+      }
+
+      const response = await userClient.get(http, '/forgot_password', clientOptions);
```

Comment:
This doesn't need to change at all, but it feels like the request parameters are buried a little bit here for me, although they seem like the most important part 🤔 

Maybe you hate the below syntax more... definitely doesn't need to change, just the readers eyes 👍 
```JS
const response = await userClient.get(http, '/forgot_password',{...options, params});
```

Reply:
👍 My reasoning was that if we ever add the ability to add params in the options object, we'll override them. But that is not really reasonable to think about now and like your argument better. Changed